bindings/bindingtester/spec/bindingApiTester.md

```diff
@@ -132,152 +132,152 @@ futures must apply the following rules to the result:
 
   - If the result is void (i.e. the future was just a signal of
     completion), then its value should be the byte string
-    "RESULT_NOT_PRESENT"
+    `"RESULT_NOT_PRESENT"`
 
   - If the result is from a GET operation in which no result was
     returned, then its value is to be converted to the byte string
-    "RESULT_NOT_PRESENT"
+    `"RESULT_NOT_PRESENT"`
 
-NEW_TRANSACTION
+#### NEW_TRANSACTION
 
     Creates a new transaction and stores it in the global transaction map
     under the currently used transaction name.
 
-USE_TRANSACTION
+#### USE_TRANSACTION
 
     Pop the top item off of the stack as TRANSACTION_NAME. Begin using the
     transaction stored at TRANSACTION_NAME in the transaction map for future
     operations. If no entry exists in the map for the given name, a new
     transaction should be inserted.
 
-ON_ERROR
+#### ON_ERROR
 
     Pops the top item off of the stack as ERROR_CODE. Passes ERROR_CODE in a
     language-appropriate way to the on_error method of current transaction
     object and blocks on the future. If on_error re-raises the error, bubbles
     the error out as indicated above. May optionally push a future onto the
     stack.
 
-GET (_SNAPSHOT, _DATABASE)
+#### GET (_SNAPSHOT, _DATABASE)
 
     Pops the top item off of the stack as KEY and then looks up KEY in the
     database using the get() method. May optionally push a future onto the
     stack.
 
-GET_KEY (_SNAPSHOT, _DATABASE)
+#### GET_KEY (_SNAPSHOT, _DATABASE)
 
     Pops the top four items off of the stack as KEY, OR_EQUAL, OFFSET, PREFIX
-    and then constructs a key selector. This key selector is then resolved 
-    using the get_key() method to yield RESULT. If RESULT starts with PREFIX, 
-    then RESULT is pushed onto the stack. Otherwise, if RESULT < PREFIX, PREFIX 
+    and then constructs a key selector. This key selector is then resolved
+    using the get_key() method to yield RESULT. If RESULT starts with PREFIX,
+    then RESULT is pushed onto the stack. Otherwise, if RESULT < PREFIX, PREFIX
```

Comment:
Should `RESULT` and `PREFIX` be in back ticks here? It doesn't matter if they are all in verbose blocks anyway, but if that gets changed, it feels like maybe that should as well.

Reply:
This is one of the reasons I left the descriptions in code blocks. If you think the descriptions look better when they are not rendered as code blocks, I'll wrap all variable literals/any code in backticks. I like the way it renders now, but I'll change it if you prefer it another way.